.gitignore

```diff
@@ -6,6 +6,10 @@
 .pnp.js
 .yarn/install-state.gz
 
+# editor
+.vscode
```

Comment:
I think this should be removed since we don't have any vscode plugins in use

Reply:
Tuedon kept it there cause he has some plugins it seems, it wouldn't hurt to have it there

Reply:
It's alright

Reply:
Yh